pigeon/weights/validators.py

```diff
@@ -16,6 +16,7 @@
 #
 
 from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext as _
```

Comment:
Hmm? Why?

Reply:
Because 5 lines below:

        raise ValidationError(_('File too large. Size should not exceed %d MiB.') % limit_mib)